spring-cloud-config-sample/pom.xml

```diff
@@ -56,6 +56,10 @@
 			<groupId>org.springframework.retry</groupId>
 			<artifactId>spring-retry</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
This will break tests. This module is less of a sample and more of integration tests.

Reply:
thank you for the feedback. I wanted to change the `sample` because of this in the documentation : _see the test cases for the config-client or the sample application_. That `or the sample application` is what I thought is about a "sample". Should that be removed from the doc then?  or there is a sample in some other place?